kanidm_book/src/administrivia.md

```diff
@@ -46,7 +50,8 @@ This is a simple backup of the data volume.
 Automatic backups can be generated online by a `kanidmd server` instance
 by including the `[online_backup]` section in the `server.toml`.
 This allows you to run regular backups, defined by a cron schedule, and maintain
-the number of backup versions to keep. An example is located in [examples/server.toml](../../examples/server.toml).
+the number of backup versions to keep. An example is located in 
+TODO this link is 404 [examples/server.toml](../../examples/server.toml).
```

Comment:
I think we can link to the book chapter on setting up the server now since we have the example config there?